src/app/clusters/laundry-washer-controls-server/laundry-washer-controls-delegate.h

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace LaundryWasherControls {
+
+/** @brief
+ *    Defines methods for implementing application-specific logic for the laundry washer controls cluster.
+ */
+class Delegate
+{
+public:
+    Delegate()          = default;
+    virtual ~Delegate() = default;
+
+    /**
+     * Get the list of supported spin_speed list.
```

Comment:
@crlonxp This was merged without addressing review comments (e.g. https://github.com/project-chip/connectedhomeip/pull/27503#discussion_r1271255904), even though those comments got marked resolved.

I am sorry, but I will no longer be marking your PRs as "approved if you address these comments".  All comments will need to be addressed before I can approve.

Reply:
Should I create another PR to address these remaining comments? Or should I keep modify the code I have and push?
Thank you

Reply:
> Should I create another PR to address these remaining comments? 

Yes, please, since this PR is now merged...

Reply:
I add the change in https://github.com/project-chip/connectedhomeip/pull/28264
Please review. Thank you